committeeoversightapp/migrations/0001_initial.py

```diff
@@ -0,0 +1,48 @@
+# Generated by Django 2.1.2 on 2018-10-31 20:41
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    initial = True
+
+    dependencies = [
+        ('legislative', '0008_merge_20181031_1341'),
```

Comment:
This points to a merge migration that, I believe, only resides on your local machine. (You can take a look at `opencivicdata/legislative/migrations` in your virtual environment, and I suspect you will see it there.) 

In any case, the dependency for this migration should be: https://github.com/opencivicdata/python-opencivicdata/blob/master/opencivicdata/legislative/migrations/0007_auto_20181029_1527.py

Can you look into this (erp, again)?

Reply:
I think this migration conflict concerns the hearings scraper, which points to a particular branch on python-opencivicdata: https://github.com/datamade/hearings/blob/master/requirements.txt#L4 

Let's touch base with @fgregg about whether he plans to merge that branch with master. 

In any case, I think we want the `committee-oversight` and `hearings` repos to point to the same version of python-opencivicdata (i.e., to avoid `django.db.migrations.exceptions.NodeNotFoundError`). 'Til Tuesday!

Reply:
@fgregg: as of last week there's a new version of OCD that has some migration changes: https://github.com/opencivicdata/python-opencivicdata/commit/3fcbae298b95311b1d9b02faf9a86704f1632234

I updated this `committee-hearings` app and it's causing some merge conflicts. We should make sure this app and the hearings scraper repo use the same version. What do you think?

Reply:
you need to point the app to the same python-opencividata as hearings.